packages/governance/src/noActionElectorate.js

```diff
@@ -13,35 +15,45 @@ import { makePromiseKit } from '@endo/promise-kit';
 const start = zcf => {
   const { subscriber } = makePublishKit();
 
-  const publicFacet = Far('publicFacet', {
-    getQuestionSubscriber: () => subscriber,
-    getOpenQuestions: () => {
+  const publicFacet = makeHeapFarInstance('publicFacet', ElectoratePublicI, {
+    getQuestionSubscriber() {
+      return subscriber;
+    },
+    getOpenQuestions() {
       /** @type {Handle<'Question'>[]} */
       const noQuestions = [];
       const questionsPromise = makePromiseKit();
       questionsPromise.resolve(noQuestions);
       return questionsPromise.promise;
     },
-    getName: () => 'no Action electorate',
-    getInstance: zcf.getInstance,
-    getQuestion: () => {
+    getName() {
+      return 'no Action electorate';
+    },
+    getInstance() {
+      return zcf.getInstance();
+    },
+    getQuestion() {
       throw Error(`noActionElectorate doesn't have questions.`);
     },
   });
 
-  const creatorFacet = Far('creatorFacet', {
-    getPoserInvitation: () => {
+  const creatorFacet = makeHeapFarInstance('creatorFacet', ElectorateCreatorI, {
+    getPoserInvitation() {
       return zcf.makeInvitation(() => {},
       `noActionElectorate doesn't allow posing questions`);
     },
     addQuestion() {
```

Comment:
I see from the guard declaration that this method has two mandatory parameters. I prefer to have these appear in the function signature, prefixed with underbar if not otherwise used.
```suggestion
    addQuestion(_instance, _question) {
```

Reply:
done